code/game/objects/structures/janitor.dm

```diff
@@ -1,13 +1,69 @@
 #define CART_HAS_MINIMUM_REAGENT_VOLUME !(reagents.total_volume < 1)
 
-/obj/structure/janitorialcart
-	name = "janitorial cart"
-	desc = "This is the alpha and omega of sanitation."
+/obj/structure/mopbucket
```

Comment:
Should be `mop_bucket`

Reply:
Applied with https://github.com/tgstation/tgstation/pull/71094/commits/f8680eb217d8ab616b5a67bb03285da875b53d45